packages/slice-machine/server/src/api/slices/rename.ts

```diff
@@ -70,14 +70,29 @@ export async function renameSlice(req: {
     `${CustomPaths(env.cwd).library(libName).value()}/${newSliceName}`
```

Comment:
do `/` paths work on windows?
maybe `path.join` would be better?

Reply:
I believe they're equivalent in Windows and code editors do accept /

Reply:
but happy to make the change